simbad/rotsearch/amore_search.py

```diff
@@ -224,6 +226,9 @@ def run(self, models_dir, nproc=2, shres=3.0, pklim=0.5, npic=50,
         self._search_results = results
         shutil.rmtree(script_log_dir)
 
+        if os.path.isdir(os.path.join(self.work_dir, 'tmp')):
+            shutil.rmtree(os.path.join(self.work_dir, 'tmp'))
```

Comment:
Would you be able to define `os.path.join(self.work_dir, 'tmp')` above and use the variable down here?

Reply:
Addressed in 3dacb91